doc: clarification for maxBuffer and Unicode output

jasnell opened the pull request:
### Pull Request check-list

- [x] Does `make -j8 test` (UNIX) or `vcbuild test nosign` (Windows) pass with
  this change (including linting)?
- [x] Is the commit message formatted according to [CONTRIBUTING.md][0]?
- [x] If this change fixes a bug (or a performance problem), is a regression
  test (or a benchmark) included?
- [x] Is a documentation update included (if this change modifies
  existing APIs, or introduces new ones)?

### Affected core subsystem(s)

doc

### Description of change

Clarify caveats on `maxBuffer` with regards to Unicode output.

Refs: https://github.com/nodejs/node/issues/1901

bnoordhuis commented:
TBH, it seems kind of pointless to me - most child processes will simply ignore it - and it's already perfectly possible to set an environment variable.  I don't really see why this should be in core.

jasnell commented:
Fair enough. What do you think about the docs change, however?

bnoordhuis commented:
The change to the documentation LGTM if you remove the reference to `NODE_CHILD_MAX_BUFFER`.

jasnell commented:
@bnoordhuis ... updated so that it's just the documentation changes. Also cleaned up some errant line-wrapping issues and end-of-line-whitespace. PTAL

bnoordhuis commented:
LGTM

benjamingr commented:
LGTM

jasnell commented:
Landed in ad2df3a

TheAlphaNerd commented:
@jasnell ltss?

TheAlphaNerd commented:
@jasnell lts?

jasnell commented:
+1
On Apr 20, 2016 11:33 PM, "Myles Borins" <notifications@github.com> wrote:

> @jasnell <https://github.com/jasnell> lts?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6030#issuecomment-212767041>
>

TheAlphaNerd commented:
@jasnell not landing cleanly, would you like to backport?

jasnell commented:
will do

TheAlphaNerd commented:
ping @jasnell

jasnell commented:
Thanks for the reminder, will do this week.

TheAlphaNerd commented:
@jasnell added don't land. Feel free to still open a backport PR
